install/frigate-install.sh

```diff
@@ -0,0 +1,181 @@
+$STD ln -svf /usr/local/go2rtc/bin/go2rtc /usr/local/bin/go2rtc
+msg_ok "Installed go2rtc"
+
+if [[ "$CTTYPE" == "0" ]]; then
+  msg_info "Setting Up Hardware Acceleration"
+  $STD apt-get -y install {va-driver-all,ocl-icd-libopencl1,intel-opencl-icd,vainfo,intel-gpu-tools}
+  chgrp video /dev/dri
+  chmod 755 /dev/dri
+  chmod 660 /dev/dri/*
+  $STD adduser $(id -u -n) video
+  $STD adduser $(id -u -n) render
+  msg_ok "Set Up Hardware Acceleration"
+fi
+
+RELEASE=$(curl -s https://api.github.com/repos/blakeblackshear/frigate/releases/latest | grep -o '"tag_name": *"[^"]*"' | cut -d '"' -f 4)
+msg_info "Installing Frigate $RELEASE (Perseverance)"
+cd ~
+mkdir -p /opt/frigate/models
+wget -q https://github.com/blakeblackshear/frigate/archive/refs/tags/${RELEASE}.tar.gz -O frigate.tar.gz
+tar -xzf frigate.tar.gz -C /opt/frigate --strip-components 1
+rm -rf frigate.tar.gz
+cd /opt/frigate
+$STD pip3 wheel --wheel-dir=/wheels -r /opt/frigate/docker/main/requirements-wheels.txt
+cp -a /opt/frigate/docker/main/rootfs/. /
+export TARGETARCH="amd64"
+echo 'libc6 libraries/restart-without-asking boolean true' | debconf-set-selections
+$STD /opt/frigate/docker/main/install_deps.sh
+$STD ln -svf /usr/lib/btbn-ffmpeg/bin/ffmpeg /usr/local/bin/ffmpeg
+$STD ln -svf /usr/lib/btbn-ffmpeg/bin/ffprobe /usr/local/bin/ffprobe
+$STD pip3 install -U /wheels/*.whl
+ldconfig
+$STD pip3 install -r /opt/frigate/docker/main/requirements-dev.txt
+$STD /opt/frigate/.devcontainer/initialize.sh
+$STD make version
+cd /opt/frigate/web
+$STD npm install
+$STD npm run build
+cp -r /opt/frigate/web/dist/* /opt/frigate/web/
+cp -r /opt/frigate/config/. /config
+sed -i '/^s6-svc -O \.$/s/^/#/' /opt/frigate/docker/main/rootfs/etc/s6-overlay/s6-rc.d/frigate/run
+msg_ok "Installed Frigate $RELEASE"
+
+msg_info "Installing Object Detection Models (Resilience)"
+$STD pip install -r /opt/frigate/docker/main/requirements-ov.txt
+cd /opt/frigate/models
+export ENABLE_ANALYTICS=NO
+$STD /usr/local/bin/omz_downloader --name ssdlite_mobilenet_v2
+cd ..
+export CCACHE_DIR=/root/.ccache
+export CCACHE_MAXSIZE=2G
+wget -q https://github.com/libusb/libusb/archive/v1.0.26.zip
+unzip -q v1.0.26.zip
+cd libusb-1.0.26
+$STD ./bootstrap.sh
+$STD ./configure --disable-udev --enable-shared
+$STD make -j $(nproc --all)
+cd /opt/frigate/libusb-1.0.26/libusb
+mkdir -p /usr/local/lib
+$STD /bin/bash ../libtool  --mode=install /usr/bin/install -c libusb-1.0.la '/usr/local/lib'
+mkdir -p /usr/local/include/libusb-1.0
+$STD /usr/bin/install -c -m 644 libusb.h '/usr/local/include/libusb-1.0'
+ldconfig
+cd ~
+wget -qO edgetpu_model.tflite https://github.com/google-coral/test_data/raw/release-frogfish/ssdlite_mobiledet_coco_qat_postprocess_edgetpu.tflite
+wget -qO cpu_model.tflite https://github.com/google-coral/test_data/raw/release-frogfish/ssdlite_mobiledet_coco_qat_postprocess.tflite
+cp /opt/frigate/labelmap.txt /labelmap.txt
+cp -r /opt/frigate/models/public/ssdlite_mobilenet_v2 openvino-model
```

Comment:
@remz1337 

ssdlite_mobilenet_v2.xml is not present at the end of the install. 

this prevent the openvino detector to work (https://docs.frigate.video/configuration/object_detectors#openvino-detector)

Reply:
File is generated during model conversion (Python). It is there in my LXC. I suggest you open an issue and we can track it there. Try to reinstall the LXC in verbose mode (advanced setting) so I can take a look at your logs.